openff/evaluator/backends/dask.py

```diff
@@ -746,13 +746,13 @@ def __init__(
         >>> # ten workers with the requested resources depending on the calculation load.
         >>> from openff.evaluator.backends.dask import DaskSLURMBackend
         >>>
-        >>> pbs_backend = DaskPBSBackend(minimum_number_of_workers=1,
+        >>> slurm_backend = DaskSLURMBackend(minimum_number_of_workers=1,
         >>>                              maximum_number_of_workers=10,
         >>>                              resources_per_worker=resources,
         >>>                              queue_name='gpuqueue',
         >>>                              setup_script_commands=setup_script_commands)
         """
-
+        resources_per_worker._wallclock_time_limit = resources_per_worker.wallclock_time_limit + ":00"
```

Comment:
What if we add an `if` statement here to check if the user has already specified the wallclock in `HH:MM:SS` format and only add `:00` if they have not. What do you think?

Reply:
I think this way would serve both purposes of keeping current functionality for LSF, and also not changing the prior definition of evaluator's [documented walltime variable](https://openff-evaluator.readthedocs.io/en/stable/api/generated/openff.evaluator.backends.QueueWorkerResources.html?highlight=wallclock_time_limit#openff.evaluator.backends.QueueWorkerResources.wallclock_time_limit). 
I actually went by [this guide from slurm](https://slurm.schedmd.com/rosetta.pdf) and changed the `wallclock_time_limit` variable in my first commit but it looks like from [IBM's](https://www.ibm.com/docs/en/spectrum-lsf/10.1.0?topic=o-w-1) and [Lilac's](https://hpcdocs.mskcc.org/display/CLUS/LSF+bsub+flags) documentation `HH:MM:SS` is invalid for LSF jobs. So it's better if we restrict user input to `HH:MM` and convert in the backend for slurm.

Reply:
Interesting, it looks like I haven't been following guidelines in the documentation at all :((. Looking at my scripts, I have been specifying the wallclock in `HH:MM:SS` format when running jobs in both PBS and SLURM environments (never tried LFS). Evaluator will let this pass since it is not doing an exact string matching. I guess if we really want to restrict users to use `HH:MM` format we may need to change the pattern below to `re.compile(r"^\d\d:\d\d$")`. I don't have a strong opinion on this, what do you think @pavankum?

https://github.com/openforcefield/openff-evaluator/blob/5c6f65d157627e750af76d15f55572f3918d036b/openff/evaluator/backends/backends.py#L168-L169

Reply:
Ahhh, this would be a non-issue then :anguished:. Grad student in the lab was complaining that anything out of format was throwing an error and that's why I jumped in to change it. I checked it now and it does take `HH:MM:SS` without any complaint!! 
Thank you for mentioning that! This may well be a small documentation update then "use `HH:MM` for LSF and `HH:MM:SS` for PBS/SLURM"

Reply:
A small documentation update sounds good 👍